package.json

```diff
@@ -105,13 +105,16 @@
     "lit": "^3.2.1"
   },
   "devDependencies": {
+    "@axe-core/playwright": "^4.10.2",
     "@changesets/changelog-github": "^0.5.0",
     "@changesets/cli": "^2.27.10",
     "@custom-elements-manifest/analyzer": "^0.10.4",
     "@eslint/eslintrc": "^3.3.1",
     "@eslint/js": "^9.28.0",
     "@figma/rest-api-spec": "^0.32.0",
     "@html-eslint/eslint-plugin": "^0.41.0",
+    "@jridgewell/trace-mapping": "^0.3.29",
```

Comment:
This and almost all the other package additions below are for the [coverage reporter](https://github.com/CrowdStrike/glide-core/pull/1011/files#diff-2ab3b29b816d4740ced71cbe68a5b326bf17ebdbcc00e35cb6591713483b0e0f) I created for Playwright. 

These packages are [extremely](https://www.npmjs.com/package/@jridgewell/trace-mapping) widely used, which bodes well. And there's a good chance that all of them are transitive dependencies of either Web Test Runner or Vitest. So we probably already depend on most of them.